content/docs/user-guide/components/reference/terrain/terrain-macro-material.md

```diff
@@ -5,23 +5,35 @@ description: 'Open 3D Engine (O3DE) Terrain Macro Material component reference.'
 weight: 100
 ---
 
-The **Terrain Terrain Macro Material** component provides a method of defining a macro-level appearance of a region of terrain.
+The **Terrain Macro Material** component applies a terrain base color texture to all terrain regions that fall within its volume.
 
 ## Usage
```

Comment:
Can you move the Usage section to be below the the Properties section? This is consistent with other component reference pages like terrain world, terrain world renderer, and others. (There are some that still have Usage at the top, but we are fixing those along the way.)

Reply:
Between Properties and EBus definitions!

Reply:
Done!